servers/src/main/java/tachyon/master/block/journal/BlockInfoEntry.java

```diff
@@ -50,4 +51,23 @@ public long getLength() {
   public JournalEntryType getType() {
     return JournalEntryType.BLOCK_INFO;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
```

Comment:
Is there any point in this check (is the following check a superset of this one)?